README.md

```diff
@@ -1,269 +1,31 @@
 # CNCF + Summer of Code
 
-The Cloud Native Computing Foundation participates in Google's Summer of Code (GSoC). CNCF is a great place to spend a summer learning, coding, participating and contributing. We are an exciting open source foundation with a vibrant community of projects, and we look forward to your application and your project ideas!
+The Cloud Native Computing Foundation participates in [Google Summer of Code](https://summerofcode.withgoogle.com/) (GSoC). CNCF is a great place to spend a summer learning, coding, participating and contributing. We are an exciting open source foundation with a vibrant community of projects, and we look forward to your application and your project ideas!
 
 ## Organization Admins
 
 If you need help with anything CNCF + Summer of code related, you can file an issue or contact one of the admins below:
 
-* Chris Aniszczyk (@caniszczyk): https://twitter.com/cra
-* Brett Preston (@bpreston)
-* Cameron Zahedi (@czahedi)
+* Chris Aniszczyk ([@caniszczyk](https://github.com/caniszczyk)): https://twitter.com/cra
+* Brett Preston ([@bpreston](https://github.com/bpreston))
```

Comment:
@nikhita please, remove Brett and add me (Ihor Dvoretskyi; github.com/idvoretskyi; twitter.com/idvoretskyi).

Thanks

Reply:
It has been added in https://github.com/cncf/soc/commit/88087edf3c369ca82de3d28389de8a1acc845be9. 👍